picard/ui/options/general.py

```diff
@@ -19,7 +19,6 @@
 
 from PyQt5 import QtCore
 from PyQt5.QtWidgets import QInputDialog
-
```

Comment:
This whiteline should stay, can you just re-add it and squash again ?

Reply:
Done, thanks :)